### DISCUSSION TITLE
Add support for SolidJS

### DISCUSSION THREAD

**Discussion by aderchox** [ionic-team/capacitor#6319] [General] [answered]
Hi, I've not used Capacitor yet (I've just got to know it exists), I wonder if SolidJS is supported too? Or if it's not, do you plan to add support?
In the main page of capacitorjs.com it names major frameworks such as React, Angular, Vue and Svelte, but Solid is missing, while it is one of the most progressing/growing UI framework projects in the community, and is already adopted by many companies and supported by major tools.
Thanks.

**Comment by mlynch** [comment, marked as answer]
Yes, Capacitor supports any web technology. It's your responsibility to provide the UI layer, of course. We have some recommendations but not all of these support Solid: https://capacitorjs.com/docs/getting-started/ui

**Comment by aderchox** [comment]
@mlynch Thank you, what do you mean "not all of these support Solid"? Your "recommendations" don't support Solid? Excuse me, the wording is a bit vague to me.

**Reply by mlynch** [reply]
Some of the UI framework recommendations in those docs are third party projects so I just can't speak to whether they support solid or not
